### DISCUSSION TITLE
请问能否增加视频截图的下载？

### DISCUSSION THREAD

**Discussion by XingXing-404** [Yuukiy/JavSP#166] [想法 | Ideas]
之前使用的软件开始收费了，无意中找到这个项目，它运行的非常完美。但是我还有一个建议，在javbd这些网站都会有一些视频截图，能否在下载封面的时候一起将这些截图爬下来？这样在emby查看的时候会更加方便

**Comment by Yuukiy** [comment]
抓取数据的时候如果有预览图其实已经抓了，只是没有下载。我先转成issue #176 了，那边好跟踪进度
